hathor/util.py

```diff
@@ -274,3 +276,19 @@ def not_none(optional: Optional[_T], message: str = 'Unexpected `None`') -> _T:
     if optional is None:
         raise AssertionError(message)
     return optional
+
+
+def random_choices(rng: Rng, options: Sequence[_T], size: int) -> List[_T]:
+    """Choose `size` randomly selected elements from `options` sequence."""
+    # XXX: explicitly convert to numpy array to prevent numpy from converting a NamedTuple (i.e.) to a numpy array too
+    _options = np.empty(len(options), dtype=object)
+    _options[:] = options
+    return list(rng.choice(_options, size, replace=False))
```

Comment:
I think including numpy in the project is error prone. We will have to remember all these details when we decide to call `choice(...)` in the future.

Reply:
Done. Removed numpy completely.